package.json

```
@@ -20,7 +20,8 @@
     "@radix-ui/react-alert-dialog": "^1.1.2",
     "@radix-ui/react-icons": "^1.3.0",
     "@radix-ui/react-label": "^2.1.0",
-    "@radix-ui/react-radio-group": "^1.2.1",
+    "@radix-ui/react-radio-group": "^1.2.0",
```

バージョンを落とす必要はないので、mainと同様のバージョンに戻してください。
また、`bun install` を実行して bun.lockb にも反映させてください。
```suggestion
    "@radix-ui/react-radio-group": "^1.2.1",
```